### PR TITLE
fix(audio): assorted fixes and improvements to LiveKit's floor manager

### PR DESCRIPTION
**Pull request by prlanzarin**
#### Summary

Do a sweep on akka-apps' audio floor managed used by the LK audio bridge and fix a
few issues:
  - Remove the possibility of cross-meeting floor collisions
  - Improve floor cooldown handling so that switches are more deterministic and reactive
  - Fixes a couple of edge cases regarding floor starvation (simultaneous speaking, stale floor entries)
  - Add a few E2E tests for floor actions (LK only)
  
#### In detail
 
- [refactor(audio): remove unused talkingStateChanges floor tracking](https://github.com/bigbluebutton/bigbluebutton/commit/a1e30a306d110fac7c1888752bee903445dfc1a6) 
  - AudioFloorManager's FloorState.talkingStateChanges has no actual usage.
Drop the field and the consequently stale cleanupOldState.
- [fix(audio): scope audio floor control state per meeting](https://github.com/bigbluebutton/bigbluebutton/commit/537136e6484c06fa1979b8e1d87c70338592ebb1) 
  - AudioFloorManager is an akka-apps-wide singleton, which may cause
collisions when handling audio floor events.
  - Turn AudioFloorManager into a per-meeting class held by LiveMeeting, so
floor grants, releases, CDs and pending queues are meeting-scoped and
releases always resolve in the holder's own meeting, and tear it down
from a MeetingActor postStop hook, which covers every termination path.
- [fix(audio): retry floor grants blocked by the floor cooldown](https://github.com/bigbluebutton/bigbluebutton/commit/0776154733e7c590c4ffdc7c83ac8ec7d00e7cc2) 
  - Replace the per-queued grant timers with a single re-scheduled grant
dispatch that always evaluates the queue head at the earliest instant a
grant can succeed (talking deadline or cooldown expiry, whichever is
later) and re-runs on every queue or floor-state change.
  - This includes the rewrite of the scheduler to use Pekko's scheduleOnce
via an internal DispatchFloorGrantsInternalMsg to the meeting actor,
handled like the other internal meeting timers, so all floor state is
inside the meeting actor lifecycle.
- [fix(audio): do not schedule the floor switch cooldown on release](https://github.com/bigbluebutton/bigbluebutton/commit/11db2d1e7d7d411a8bffea02825e573df9e62c1a) 
  - When a floor holder leaves, the release itself starts a cooldown that
defers the grant to the next pending speaker until after the cooldown
elapses. The cooldown exists to space out floor switches only - a release
event always should come after the cooldown already acted.
- [fix(audio): prune stale speaking states in floor manager](https://github.com/bigbluebutton/bigbluebutton/commit/fbbb3d24a555d0c54cc11a782eeb15d9c419d0b2) 
  - speakingStartTimes entries are removed only by explicit stop-talking or
left-voice events. A user whose events are lost (disconnect, ...) leaves an
entry behind until meeting end. Two obvious issues: queue buildup and
floor grant block for the affected user.
- [tests(audio): cover LiveKit audio floor control](https://github.com/bigbluebutton/bigbluebutton/commit/486f85968145c07a4697f05b4472fc71be8c17ff) 
  - Cover the akka floor manager in E2E tests through a few test scenarios:
    - Floor isolation across two concurrent meetings
    - No starvation when two speakers unmute near-simultaneously
    (both must eventually be granted)
    - Granting of the floor to a remaining active speaker when a floor
    holder leaves the meeting.
  - See commit log for implementation details.
  
### Closes Issue(s)

None

### CODE REVIEW

File: akka-bbb-apps/src/main/scala/org/bigbluebutton/core/apps/voice/AudioFloorManager.scala [hunk 1]
```diff
@@ -1,106 +1,144 @@
...
-    state = FloorState()
+    if (stale.nonEmpty) {
+      logFloorEvent("none", "stale_speaking_state_pruned", Map(
+        "users" -> stale.mkString(",")
+      ))
+      state = state.copy(speakingStartTimes = state.speakingStartTimes -- stale)
+      if (pendingFloors.exists(pending => stale.contains(pending.userId))) {
+        pendingFloors.filterInPlace(pending => !stale.contains(pending.userId))
+        dispatchPendingGrants(liveMeeting, outGW)
+      }
```
[the remark this answers is not in the document]
Comment (prlanzarin) [reply akka-bbb-apps/src/main/scala/org/bigbluebutton/core/apps/voice/AudioFloorManager.scala:84]: This is intentional design. Floor is sticky unless a new floor supersedes it or staleness ensues - for reconnect persistence etc. Same thing as in the SFU/FS floor logic.

**Review by germanocaumo — APPROVED** [review]
Reviewed and tested locally, look good
